src/workers/parser/mapOriginalExpression.js

```diff
@@ -10,11 +10,14 @@ import { buildScopeList } from "./getScopes";
 import generate from "@babel/generator";
 import * as t from "@babel/types";
 
-// NOTE: this will only work if we are replacing an original identifier
+// NOTE: this will only work
+// if we are replacing an original identifier or an object property
 function replaceNode(ancestors, node) {
   const ancestor = ancestors[ancestors.length - 1];
 
-  if (typeof ancestor.index === "number") {
+  if (t.isObjectProperty(ancestor.node)) {
```

Comment:
I don't think this approach is the right way to go about this.

There should already be a `node.value` in the AST, AFAIK, but it looks like we are failing to find it, I assume because it has the same location as the existing `node.key`.

It _might_ actually be enough to do
```
  t.traverse(ast, (node, ancestors) => {
    if (!t.isIdentifier(node) && !t.isThisExpression(node)) {
      return;
    }

    const ancestor = ancestors[ancestors.length - 1];
    if (ancestor.node.type === "ObjectProperty" && ancestor.key !== "value") {
      return;
    }

    // ...
```